posts/2025-03-03-Rust-participates-in-GSoC-2025.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Rust participates in Google Summer of Code 2025"
+author: Jakub Beránek, Jack Huey and Paul Lenz
+---
+
+We're writing this blog post to announce that the Rust Project will again be participating in [Google Summer of Code (GSoC) 2025][gsoc]. If you're not eligible or interested in participating in GSoC, then most of this post likely isn't relevant to you; if you are, this should contain some useful information and links.
```

Comment:
Since a lot of this is the same as last year's, I wonder if it's worth explicitly mentioning and linking that post? (Otherwise, I often feel like it's easy to get in the "haven't I read this already" feeling)